src/OrchardCore.Modules/OrchardCore.Users/Views/Account/Login.cshtml

```diff
@@ -81,3 +86,23 @@
         </div>
     }
 </div>
+<script type="text/javascript" at="Foot">
```

Comment:
While we have a dependency on jQuery, it will be much simpler if we use it

Reply:
Well I explained why I did not use it. Now this is just a question of taste at this point. I generally prefer not having to rely on JQuery for everything. If we do this then we should also move this javascript code in the OrchardCore.Users module as a script resource for future extensibility purposes. Though, I'm not sure this is needed unless someone convince me of the opposite. It's just a visual component ...

Reply:
No problem, but while there are some UI components are using jQuery heavily seems it needs difficult to rid off the dependency or we may need a time